Source/WebInspectorUI/UserInterface/Models/CSSCompletions.js

```diff
@@ -131,6 +131,19 @@ WI.CSSCompletions = class CSSCompletions
         return suffix;
     }
 
+    static getCompletionText(completion)
+    {
+        console.assert(typeof completion === "string" || completion instanceof WI.QueryResult, completion);
+
+        if (typeof completion === "string")
+            return completion;
+
+        if (completion instanceof WI.QueryResult)
+            return completion.value;
+
+        return "";
```

Comment:
Nit: You could just `console.assert(false, "explanation")` at the end instead of writing each condition twice. (I know you are just moving the code in this patch, but might be a good opportunity to simplify.